api/src/Tests/Cases/DeadlineTest.php

```diff
@@ -59,12 +65,25 @@ protected function setUp(): void
 
         $this->target = $target;
 
+        $when = date('Y-m-d', strtotime('+1998 years'));
+        $to = date('Y-m-d', strtotime('+2002 years'));
+        $this->cycle = $this->runSuccessJsonRequest('POST', '/cycle', null, ['From' => $when, 'To' => $to], 201);
+        $when = date('Y-m-d', strtotime('+1999 years'));
+        $to = date('Y-m-d', strtotime('+1999 years'));
+        $this->event = $this->runSuccessJsonRequest('POST', '/event', null, ['From' => $when, 'To' => $to, 'Name' => 'PHPTest-a-con'], 201);
```

Comment:
So, this is an example of what I mean by "just use the database". We're not testing the Event API here, but the Deadline API. So instead of using the Event API, just us an Atlas\Query\Insert to stash the objects and an Atlas\Query\Delete to tear them down later.